packages/desktop-client/src/queries/pagedQuery.test.ts

```
@@ -618,4 +618,100 @@ describe('pagedQuery', () => {
+    for (let i = 0; i < 400; i++) {
+      data.push({ id: i });
+    }
+
+    // Server that blocks every page query until we release it, so the test
+    // controls exactly when each in-flight request resolves.
+    const pending = [];
+    mockServer({
+      send: async (name, args) => {
+        switch (name) {
+          case 'query':
+            const query = args;
```

Biome (2.5.5)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)